### PR TITLE
Add LiveCode

### PR DESCRIPTION
**Pull request by fischl-ediservice**
## Adding a language

- [x] The code displays "Hello World"
- [x] I have updated the readme to include the new language
- [x] I have incremented the language count in the readme

### CODE REVIEW

File: l/Livecode [hunk 1]
```diff
@@ -0,0 +1 @@
+answer "Hello World!"
```
Comment (MrBrain295) [inline l/Livecode:1]: It needs to print “Hello World”.

**Review by MrBrain295 — COMMENTED** [review]
You also need to update the readme and provide a link to the languages website or code.

**Comment by MrBrain295** [comment]
Is it Livecode or Livescript?

**Comment by Richienb** [comment]
It appears to be called **LiveCode**, which the capitalisation should reflect in this PR.

https://lessons.livecode.com/m/2592/l/21009-hello-world

**Review by Richienb — CHANGES_REQUESTED** [review]
The capitalisation of the filename should also be changed

**Comment by ThePeeps191** [comment]
I have added these changes in #1246

**Pull request closed without merging**